projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -1322,6 +1322,23 @@ private boolean getAnswer(
     return parameters;
   }
 
+  /**
+   * Get information of the container(first element in {@code parameters}) return true if
```

Comment:
Please clean up this javadoc. Space before `(` and I can't tell if this is 1 or 2 sentences, etc. Use punctuation.